website/views.py

```diff
@@ -4744,3 +4746,92 @@ def blt_tomato(request):
         project["funding_hyperlinks"] = funding_link
 
     return render(request, "blt_tomato.html", {"projects": data})
+
+
+@login_required
+def vote_suggestions(request):
+    if request.method == "POST":
+        user = request.user.username
+        data = json.loads(request.body)
+        suggestion_id = data.get("suggestion_id")
+        suggestion = Suggestion.objects.get(id=suggestion_id)
+        up_vote = data.get("up_vote")
+        down_vote = data.get("down_vote")
+        liked = SuggestionLikes.objects.filter(user=user, suggestion=suggestion).exists()
```

Comment:
The name is misleading -- this one can be either upvote/downvote action.
```suggestion
        voted = SuggestionLikes.objects.filter(user=user, suggestion=suggestion).exists()
```